mod_form.php

```diff
@@ -100,35 +106,38 @@ public function definition() {
             }
         }
 
+        $inputstyle = array('style' => 'width: 399px');
+
         // Form start.
         $mform =& $this->_form;
         $mform->addElement('hidden', 'course', $id);
         $mform->addElement('header', 'general', get_string('general', 'form'));
-        $mform->addElement('static', 'overview', get_string('overview', 'accredible'),
-            get_string('activitydescription', 'accredible'));
-        if ($alreadyexists) {
-            $mform->addElement('static', 'additionalactivitiesone', '', get_string('additionalactivitiesone', 'accredible'));
-        }
-        $mform->addElement('text', 'name', get_string('activityname', 'accredible'), array('style' => 'width: 399px'));
+
+        $mform->addElement('text', 'name', get_string('activityname', 'accredible'), $inputstyle);
         $mform->addRule('name', null, 'required', null, 'client');
         $mform->setType('name', PARAM_TEXT);
         $mform->setDefault('name', $course->fullname);
 
         if ($alreadyexists) {
-            $mform->addElement('static', 'additionalactivitiestwo', '', get_string('additionalactivitiestwo', 'accredible'));
+            $mform->addElement('static', 'additionalactivitiesone', '', get_string('additionalactivitiesone', 'accredible'));
         }
 
-        // If we're updating and have a group then let the issuer choose to edit this.
+        // Load available groups.
+        $localgroups = new groups();
```

Comment:
It was defined like this before because of the class is defined in `local/groups`. I have renamed this to be `groupsclient` and also renamed the `localcredentials` to be `credentialsclient` for consistency.